### ISSUE TITLE
Create a PSP

### ISSUE THREAD

**Issue by majst01** [majst01/csi-driver-s3#2]

**Comment by 912988434** [comment]
@majst01  hello,The deploy/kubernetes/PSP. Yaml is used to mount?

**Comment by majst01** [comment]
No, the PSP defines the required PodSecurityPolicies if the cluster is configured to enforce them.

Mounting is done with the standard k8s PVC/PV mechanisms, see: https://kubernetes.io/docs/concepts/storage/persistent-volumes/

**Comment by 912988434** [comment]
> 不，如果集群配置为强制实施，则PSP定义必需的PodSecurityPolicies。
> 
> 使用标准的k8s PVC / PV机制完成安装，请参见：[https](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) ://kubernetes.io/docs/concepts/storage/persistent-volumes/[](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)

I see. Thank you ！
